### ISSUE TITLE
Lollipop Chainsaw [BLUS30917] - Audio cuts out after a random period of time.

### ISSUE THREAD

**Issue by Artins90** [RPCS3/rpcs3#9524]
While playing Lollipop Chainsaw the audio stops completely after a random period of time.
Sometimes it happens after a few minutes, other times after 40 minutes or more. 
It doesn't seem to be linked to any specific action in the game; it can happen while browsing the in-game menus, during gameplay or at a loading screen.

Things I have tried so far:
Changing SPU block size to safe.
Setting SPU threads to auto and 1.
Increasing the audio buffer to 250 ms.

OS: Windows 10 19042.685
CPU: Intel i7 7820X (old firmware and microcode, TSX force abort not applied).
GPU: RTX 2080 TI on driver 460.97

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/5755395/RPCS3.log.gz)

**Comment by Satan86** [comment]
I've seen people suggest disabling TSX fixes the issue

**Comment by Artins90** [comment]
I tried the game again with TSX set to disabled.
I was able to complete 2 chapters without the audio cutting off for about 2 hours, it's probably safe to say that TSX was causing the issue.
Maybe it might be worth mentioning that TSX can cause audio issues on the [wiki page](https://wiki.rpcs3.net/index.php?title=Lollipop_Chainsaw) for this game?

I am not sure if I should close this issue, thank you for proving the right suggestion.
